.gitignore

```diff
@@ -39,3 +39,7 @@ coverage.xml
 .vscode/*
 
 .config/r5py.yml
+*.ipynb
```

Comment:
please don’t exclude all ipynb and all csv files

Reply:
true good point, will remote this